src/containers/FooterContainer/FooterContainer.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { useConfiguration } from '../../context/configurationContext';
+import loadable from '@loadable/component';
+
+const SectionBuilder = loadable(() => import('../PageBuilder/PageBuilder'), {
+  resolveComponent: components => components.SectionBuilder,
+});
+
+const FooterContainer = () => {
+  const { footer } = useConfiguration();
+
+  // The footer asset does not specify sectionId or sectionType. However, the SectionBuilder
+  // expects sectionId and sectionType in order to identify the section. I suggest that we
+  // add those attributes here before passing the asset to SectionBuilder.
```

Comment:
Hmm. Don't leave "I suggest" to code comments.

We could ask for a change to the footer asset schema or just describe that we add those attributes here.

However, it would feel weird to ask footer id from the operator in content/footer.json, which leads to the situation that the id would need to be hard-coded somewhere. So, I agree that this is as good a place as the asset schema file for this (rather odd) "section" config.

Reply:
Comment reworded 👍